src/components/CharacterLayout/CharacterLayout.js

```diff
@@ -0,0 +1,43 @@
+import React, {useEffect, useState} from 'react';
+import {ActivityIndicator, FlatList, View} from 'react-native';
+
+import Input from '../Input';
+import styles from './CharacterLayout.styles';
+import CircleCharacter from '../CircleCharacter';
+import useFetch from '../../hooks/useFetch';
+
+
+const CharacterLayout = () => {
+  const [search, setSearch] = useState('');
+  const {loading, error, data, setParams, refetch} = useFetch('characters');
```

Comment:
Bu logic katmanını ayırmak için Layout yapısı kullanıyoruz ama logic kısmı buraya taşınmış. Yani yapmak istediğimizin tam tersi yapılmış. Bunu `Character.js` içinde yapıp bu component'e sadece datalar aktarılacaktı. Yanlış yapmışsınız.